WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift

```diff
@@ -402,13 +418,10 @@ public class BlockEditorScreen: ScreenObject {
                     .selectAlbum(atIndex: 0)
     }
 
-    private func confirmPost(button: XCUIElement) throws {
-        if FancyAlertComponent.isLoaded() {
-            try FancyAlertComponent().acceptAlert()
```

Comment:
I don't know which FancyAlert we could expect here. I've run all the editor tests several times and haven't seen it. We are saving some time here but it doesn't impact the target issue as it comes before tapping the `Schedule Now` button.